### ISSUE TITLE
Rename "Matchmaking" to make it clearer?

### ISSUE THREAD

**Issue by vgaming** [wesnoth/wesnoth#8148]
### Describe the desired feature

Is it possible to rename the current "Matchmaking" tab to a more elaborate/clear name? And/or add a hover help that explains what it is?

Context: in the current UI, the main Lobby contains two tabs: `General Lobby` and `Matchmaking`. The General Lobby contains human-created games, while the Matchmaking tab contains "bot-started" games, such as AfterBot games on 1.16.

The problem with the current approach is that it might not be clear. Maybe it's just my lack of understanding, but I completely failed to figure out what Matchmaking is, until chatting with people on Discord.

Notably:
- Wesnoth's "Matchmaking" is NOT actually a [server browser](https://en.wikipedia.org/wiki/Matchmaking_(video_games)#Server_browsers) functionality. This is because you don't connect to any other additional servers once you connect to the Wesnoth server. All the traffic is centralized, so the concept doesn't apply.
- Wesnoth's Matchmaking works differently from e.g. Chess or Go. Instead of asking to have a game and being automatically connected to opponents of a similar rank, wesnoth's Matchmaking process is more manual. You can connect to games where you already know the opponent, fox example.
- Without an online Bot, it's impossible to interact with this functionality at all.

Some alternative names:
- "Server-created games"
- "Bot-created games"

P.S. See also [this discussion](https://github.com/wesnoth/wesnoth/issues/7056) to understand what this feature is about and how it was created.

**Comment by Pentarctagon** [comment]
@stevecotton @Discontinuum FYI

**Comment by Discontinuum** [comment]
It was named "Matchmaking" because bots host games basically to create matchmaking, as Wesnoth lacks any automatic matchmaking of its own. The original variant sounded too technical to my taste, not telling enough for a common player

**Comment by stevecotton** [comment]
At the moment, I think "Matchmaking" is the best of the suggested names.

**Comment by soliton-** [comment]
Is this related to what matchmaking means in any other game?

If this is an area for automatically hosted games then it should say that.

**Comment by vgaming** [comment]
From my personal point of view, I was just very confused when I saw the tab. An on-hover explanation or any any other means to let the user understand what it means would be good from my perspective, too

E.g. I only want to avoid a situation when a person doesn't know what this tab means and gets confused without any way to get help.

_We_ know that wesnoth doesn't have automatic matchmaking and therefore we have to make alternative choices, but _the player_ does not know that.

Thoughts?

**Comment by vgaming** [comment]
* edited --^

**Comment by Discontinuum** [comment]
I think it looked confusing primarily because the lobby of 1.17 is empty right now

**Comment by vgaming** [comment]
@Discontinuum to be specific, the lobby didn't have AfterBot. The rest was there: I've used 3 accounts to create up to 2-3 games to try to figure it out

**Comment by Hejnewar** [comment]
> we know that wesnoth doesn't have automatic matchmaking

But we should have one

**Comment by knyghtmare** [comment]
> But we should have one

_adds plus one vote_

**Comment by Discontinuum** [comment]
@vgaming 

>  the lobby didn't have AfterBo

Yeah, it's running on 1.16 currently. I think I'll add some test bot to the 1.17 lobby to see how it looks

**Comment by soliton-** [comment]
> I think it looked confusing primarily because the lobby of 1.17 is empty right now

If the tab title is confusing unless there are games running that display in the tab it sounds like the actual info you gather by extrapolating from the displayed games and not from the meaning of the tab title. It would be better to have an actually meaningful tab title.

**Comment by vgaming** [comment]
It's slightly even more complex I'm afraid. If I understand the intentions behind creating Matchmaking (https://github.com/wesnoth/wesnoth/issues/7056) correctly, then we want to specifically split out bot-initiated games. So even if we'd get some form of Matchmaking, we may still want to keep a separate tab or... supersede the bot with this new Matchmaking? Basically, the question of what would happen to AfterBot is not clear to me.

I think we should also create a separate issue for true Matchmaking BTW. The current issue, while worded abstractly, is about AfterBot and similar bots that de-facto exist today..

**Comment by soliton-** [comment]
Discussing actual matchmaking here nicely shows how confusing the current wording is.

The previous titles of player and bot hosted were simple and clear. Now people will think there is actual matchmaking functionality and get confused until they figure out that it's not about matchmaking.

**Comment by Discontinuum** [comment]
> it's not about matchmaking

In a certain sense it is, isn't it?

**Comment by soliton-** [comment]
> > it's not about matchmaking
> 
> In a certain sense it is, isn't it?

Name a game that implements matchmaking like this.

Matchmaking is joining some kind of queue where players from the same queue are matched up based on whatever and then a game is made with those matched up players.

I doubt anyone will guess that in wesnoth it means seeing a list bot created games.

**Comment by Discontinuum** [comment]
> a list of bot created games.

A list of games that serve as that queue for players to join and be matched up

**Comment by Pentarctagon** [comment]
It sounds like the difference is:
* the queues are not provided by Wesnoth itself
* the games themselves are the queue you need to join

**Comment by soliton-** [comment]
> > a list of bot created games.
> 
> A list of games that serve as that queue for players to join and be matched up

By that definition both tabs do that...

**Comment by stevecotton** [comment]
> The previous titles of player and bot hosted were simple and clear.
> ...
> I doubt anyone will guess that in wesnoth it means seeing a list bot created games.

Bot-hosted and bot-created are actually different things, depending on whether the bot is still the host when the game starts. However, both are low-level details.

A UI label should be about what it means to the player, rather than about the details of the implementation. So far I haven't seen a suggestion in this thread for a name that feels more meaningful to the player than "matchmaking".

**Comment by soliton-** [comment]
> So far I haven't seen a suggestion in this thread for a name that feels more meaningful to the player than "matchmaking".

What meaning is that?

**Comment by stevecotton** [comment]
Although the English text for 1.18 is frozen, 8660561ad980c6133a58e5e35d465af06e238c97 includes a po hint referring to the discussion here.

The German translation is going for "Bot-initialised games" rather than "Bot-hosted games".
